### ISSUE TITLE
Can predictions be bounded?

### ISSUE THREAD

**Issue by bizbizzz** [dotnet/machinelearning#5414]
**Describe Bug**
Predicted values are outside of expected range. Is there a way to bound the predictions? Or set to a minimum value?

**To Reproduce**
Generate regression model using CLI and play with inputs

**Comment by frank-dong-ms** [comment]
Yeah, checked our current implementation this is not supported yet so mark as feature request (enhancement).

**Comment by rosebyte** [comment]
Triage: Standard regression produces an unconstrained real-valued score, so predictions can fall outside the observed label range when the model extrapolates. [ML.NET](http://ml.net/) does not currently provide a built-in, serialisable pipeline component for applying minimum or maximum bounds to regression output.

Callers can enforce a business-level output constraint by applying `Math.Clamp`, `Math.Max`, or `Math.Min` to the predicted `Score`, or by adding equivalent custom post-processing. If the bound is part of the statistical problem rather than merely an output policy, a suitable label transformation, link function, or constrained trainer may be preferable because hard clipping can hide poor extrapolation and introduce bias.
